docs/content/docs/configuration/configuration.md

```diff
@@ -278,6 +279,15 @@ telemetry:
 The `dependencies-v2` experimental flag is not yet implemented.
 When it is completed, it is used to activate the features from [PEP003 - Advanced Dependencies](https://github.com/getporter/proposals/blob/main/pep/003-advanced-dependencies.md).
 
+### Full control Dockerfile
+
+The `full-control-dockerfile` experimental flag disables all Dockerfile generation when building bundles.
+When enabled Porter will use the file referenced by `dockerfile` when building the invocation image without modifying it in any way.
```

Comment:
It could be little unclear that you mean the `dockerfile` field in the manifest file. The first time I read it, I was thinking about a flag on the command line. It would be great to be a little more specific on it being the field in the manifest file.

Reply:
Good point, I'll fix that.